### PR TITLE
Fix sidebar for `version-main` version of docs

### PR DESCRIPTION
**Pull request by IsAmrish**
Signed-off-by: isamrish <askmaurya48@gmail.com>

This commit will fix
- sidebar of `version-main` of the docs

Before:

Screenshot 2021-08-17 at 1 30 04 PM

After:
Screenshot 2021-08-17 at 1 36 42 PM

### CODE REVIEW

**Review by rakeshPRaghu — APPROVED** [review]
LGTM

File: docs/i18n/en/docusaurus-plugin-content-docs/current.json [hunk 1]
```diff
@@ -50,5 +50,73 @@
...
     "message": "cstor",
     "description": "The label for category cstor in sidebar tutorialSidebar"
+  },
+  "version.label": {
+    "message": "Version Main",
+    "description": "The label for version current"
+  },
+  "sidebar.sidebar.category.Introduction": {
+    "message": "Introduction",
+    "description": "The label for category Introduction in sidebar sidebar"
```
Comment (AVRahul) [inline docs/i18n/en/docusaurus-plugin-content-docs/current.json:60]: @IsAmrish why the **sidebar** word is repeated twice in every description ??

**Review by Pallavi-PH — APPROVED** [review]

File: docs/i18n/en/docusaurus-plugin-content-docs/current.json (hunk 1 above)
Comment (IsAmrish) [reply docs/i18n/en/docusaurus-plugin-content-docs/current.json:60]: this is auto-generated @AVRahul

**Review by AVRahul — APPROVED** [review]

File: docs/i18n/en/docusaurus-plugin-content-docs/current.json (hunk 1 above)
Comment (IsAmrish) [reply docs/i18n/en/docusaurus-plugin-content-docs/current.json:60]: We are trying to support internationalization (support multiple languages )  but currently only supporting the `english`

Comment (AVRahul) [reply docs/i18n/en/docusaurus-plugin-content-docs/current.json:60]: ok

**Pull request merged**